### DISCUSSION TITLE
Does quickwit support OpenSearch's `join_field`?

### DISCUSSION THREAD

**Discussion by ibotty** [quickwit-oss/quickwit#5031] [General]
If not is it somehow in scope?  It would make some things way easier.

See https://opster.com/guides/opensearch/opensearch-data-architecture/how-to-model-relationships-between-documents-in-opensearch-using-join/#How-to-use-the-join-field-type for an example.

**Comment by fulmicoton** [comment]
Not in scope at the moment unfortunately.

Can you share more info on your use case for this feature?
